### PR TITLE
NAS-110840 / 21.08 / Add typing to EntityWizard configuration

### PR DESCRIPTION
**Pull request by AlexKarpov98**

### CODE REVIEW

File: src/app/pages/data-protection/replication/replication-wizard/replication-wizard.component.ts [hunk 1]
```diff
@@ -56,7 +57,7 @@ export class ReplicationWizardComponent {
   saveSubmitText = T('START REPLICATION');
 
   protected entityWizard: any;
```
Comment (denysbutenko) [inline src/app/pages/data-protection/replication/replication-wizard/replication-wizard.component.ts:59]: seems like should be `EntityWizardComponent` type

**Review by undsoft — APPROVED** [review]
Looks good, thanks.

File: src/app/pages/data-protection/replication/replication-wizard/replication-wizard.component.ts (hunk 1 above)
Comment (AlexKarpov98) [reply src/app/pages/data-protection/replication/replication-wizard/replication-wizard.component.ts]: @denysbutenko Updated.

**Review by denysbutenko — APPROVED** [review]
LGTM

**Pull request merged**